packages/rich-text/src/component/index.native.js

```diff
@@ -961,13 +981,32 @@ export class RichText extends Component {
 											onClick={
 												this.handleUserSuggestion
 											}
+											onLongPress={
+												this.handleSuggestionLongPress
+											}
 										/>
 									</Toolbar>
 								)
 							}
 						</BlockFormatControls>
 					</>
 				) }
+				<Picker
+					ref={ ( instance ) => ( this.suggestionPicker = instance ) }
+					options={ [
+						{
+							value: 'mention',
+							label: __( 'Mention' ),
+							icon: atSymbol,
+						},{
+							value: 'xpost',
+							label: __( 'Xpost' ),
+							icon: plus,
+						}
+					] }
+					onChange={ onSuggestionPickerSelect }
+					hideCancelButton
+				/>
```

Comment:
@mchowning I wasn't sure where to add this `Picker` since it's rendered outside of the component hierarchy, but it seems like anywhere will do so I chose here. Let me know if you think it belongs somewhere else.

Also, this file is huge and I've added more lines of code. Not sure if we should try to move this to a new file, but left it as-is for now.